app/models/ontology/associations_and_attributes.rb

```diff
@@ -0,0 +1,38 @@
+module Ontology::AssociationsAndAttributes
+  extend ActiveSupport::Concern
+
+  included do
+    belongs_to :language
+    belongs_to :logic, counter_cache: true
+    belongs_to :ontology_type
+    belongs_to :repository
+    belongs_to :formality_level
+
+    has_many :symbol_groups
+    has_many :alternative_iris, dependent: :destroy
+    has_many :source_mappings,
+      class_name: 'Mapping', foreign_key: 'source_id', dependent: :destroy
+    has_many :target_mappings,
+      class_name: 'Mapping', foreign_key: 'target_id', dependent: :destroy
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.